Makefile

```diff
@@ -44,7 +44,7 @@ SRCS-y += lls/main.c lls/cache.c lls/arp.c lls/nd.c
 SRCS-y += lib/mailbox.c lib/net.c lib/flow.c lib/ipip.c \
 	lib/luajit-ffi-cdata.c lib/launch.c lib/lpm.c lib/acl.c
 
-LDLIBS += $(LDIR) -Bstatic -lluajit-5.1 -Bdynamic -lm
+LDLIBS += $(LDIR) -Bstatic -lluajit-5.1 -Bdynamic -lm -lmnl
```

Comment:
Please add libmnl to the list of dependencies in `README.md`. It would be nice if you mentioned the package needed to run Gatekeeper (i.e. just the binary libmnl), and the packages to compile Gatekeeper ((i.e. the binary libmnl and the package with the headers).